tests/test_discounts.py

```diff
@@ -201,6 +201,29 @@ def test_invalid_checkout_discount_form(monkeypatch, voucher):
     assert 'voucher' in form.errors
 
 
+def test_voucher_queryset_active(voucher):
+    vouchers = Voucher.objects.all()
+    assert len(vouchers) == 1
+    active_vouchers = Voucher.objects.active(
+        today=date.today() - timedelta(days=1))
+    assert len(active_vouchers) == 0
+
+
+@patch('saleor.discount.forms.date')
```

Comment:
Why not use the `monkeypatch` fixture from pytest?

Reply:
hmmm i don't know, in other test i used patch. I can change it to monkeypatch if this is the 'desired' way.

Reply:
I went a different path. instead of mocking datetime.date.today i've changed the start_time of voucher. It is easier to understand. No mocking or monkeypatching required.
Also i split the test in two test cases.

Reply:
Now i realized that those 2 new tests is not recreating the bug because it occurs after some server up time. I will add the previous version of this test also to the suite.